configure.ac

```diff
@@ -1671,6 +1671,41 @@ if test "x${je_cv_gcc_builtin_popcountl}" = "xyes" ; then
   AC_DEFINE([JEMALLOC_INTERNAL_POPCOUNTLL], [__builtin_popcountll], [ ])
 fi
 
+dnl Configure maximum number of tcaches/arenas
+dnl The upper limit of 17 is not a technical limitation, rather, it is what our
+dnl unit tests pass at. Some tests allocate areans and never destroy them, which
+dnl means we run out of areans during tests. But we can't "support" a limit we
+dnl cannot test...
```

Comment:
Thanks. 

Suggest also documenting the lower bound of 4.

Reply:
Done